py/client/pydeephaven/ticking/table_listener.py

```diff
@@ -0,0 +1,117 @@
+from abc import ABC, abstractmethod
+from typing import Dict, Generator, List, Tuple
+import pyarrow.flight as flight
+import pydeephaven.ticking.deephaven_core as dhc
+import pydeephaven.ticking.util as tick_util
+import threading
+
+# TODO(kosak): get typing right
+ColDictType = Dict[str, Tuple[List, List[bool]]]
```

Comment:
Do we really need the 2nd list to identify the null-ness of the 1st list? In most if not all cases, None can be used and is perfect legal as a value of a list item.

Reply:
1. Indeed my comment is base on the 'wrong' type hints. It sounds like what you intended is Tuple[np.ndarray, np.ndarray]. 
2. Using numpy arrays is clearly more consistent withe the server API, but I am wondering if an Arrow array would serve us better here. After all, the client API can only provide the data of a Table in the form of a pyarrow table, so it is quite acceptable that the ticking table updates are provided as arrow data which has built-in null support and handling.